app/code/Magento/NewRelicReporting/Model/Apm/Deployments.php

```diff
@@ -15,7 +15,7 @@ class Deployments
     /**
      * API URL for New Relic deployments
      */
-    const API_URL = 'https://api.newrelic.com/deployments.xml';
+    const API_URL = 'https://api.newrelic.com/v2/applications/%s/deployments.json';
```

Comment:
Should we take value from config (app/code/Magento/NewRelicReporting/etc/config.xml) ?

Reply:
Hey @swnsma 
We do, 

```
$apiUrl = $this->config->getNewRelicApiUrl();
        if (empty($apiUrl)) {
            $this->logger->notice('New Relic API URL is blank, using fallback URL');
            $apiUrl = self::API_URL;
        }
```

I see that config.xml is exists in module and cannot be empty in this case, but also may happen that somebody already extended module and for some reason using constant. So I would leave it there